datasets/README.md

```diff
@@ -0,0 +1,41 @@
+# D3M datasets
+
+Please report issues on [MIT-LL's repository](https://gitlab.datadrivendiscovery.org/MIT-LL/d3m_data_supply/issues).
+
+## Downloading
+
+Download datasets using [git LFS](https://git-lfs.github.com/):
+
+```
+$ git lfs clone git@gitlab.datadrivendiscovery.org:d3m/datasets.git
+```
+
+Note, use `git lfs clone` instead of `git clone` because it
+is faster.
+
+This will take time but especially disk space. Currently all
+datasets are around 54 GB, but the whole directory with cloned
+repository and git metadata is around 84 GB. Running
+`git lfs prune` might help by removing old and unreferenced files.
+
+Repository is organized so that all files larger than 100 KB are
+stored in git LFS, while smaller files are managed through git
+directly. This makes cloning faster because there is no need
+to make many HTTP requests for small git LFS files which is slow.
```

Comment:
Love the documentation @orionw, thanks for explaining that!

Reply:
Haha don't thank me too much, I just added the small seed datasets to the repo and copied the d3m datasets readme.  It needs to be cleaned up a little.  I do think a readme for the datasets in the experimenter would be helpful though, I agree!  I'll work on fixing the CI later today